modules/nginxvts/collect.go

```diff
@@ -0,0 +1,85 @@
+package nginxvts
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/netdata/go.d.plugin/pkg/stm"
+	"github.com/netdata/go.d.plugin/pkg/web"
+)
+
+const vtsURLPath = "/status/format/json"
```

Comment:
There is no guarantee it is `/status`, it is user configurable location

I think we need to ask users to set url to _status_ location and add `/format/json` during HTTP call.